### PR TITLE
Add note about --match-timestamp for bisects

### PR DESCRIPTION
**Pull request by keith**

### CODE REVIEW

File: docs/DebuggingTheCompiler.md [hunk 1]
```diff
@@ -822,6 +822,13 @@ reducing SIL test cases by:
 For more information and a high level example, see:
 ./swift/utils/bug_reducer/README.md.
 
+### Syncing branches during bisects
+
+Depending on what you're bisecting it might be necessary to run the
+`update-checkout` script each time you change shas. To do this you can
+pass `--match-timestamp` to automatically checkout the right revisions
+across the repos.
```
Comment (gottesmm) [inline docs/DebuggingTheCompiler.md:830]: Can you make it clear that it syncs I think to the timestamp on the swift repo (I think that's correct, right?)? 

Assuming my memory is correct, make that change and you got my +1.

Comment (keith) [reply docs/DebuggingTheCompiler.md]: Ah yes it is, done!

**Review by gottesmm — APPROVED** [review]
LGTM

**Comment by gottesmm** [comment]
@swift-ci smoke test and merge

**Comment by keith** [comment]
Thanks!

**Pull request merged**